src/converter/one/convert/JfrToOtlp.java

```diff
@@ -257,14 +264,36 @@ public boolean equals(Object o) {
             if (!(o instanceof KeyValue)) return false;
 
             KeyValue other = (KeyValue) o;
-            return key.equals(other.key) && value.equals(other.value);
+            return keyStrindex == other.keyStrindex && value.equals(other.value);
         }
 
         @Override
         public int hashCode() {
             int result = 17;
-            result = 31 * result + key.hashCode();
+            result = 31 * result + keyStrindex;
             return 31 * result + value.hashCode();
         }
     }
+
+    private static final class IntArrayWrapper {
+        public static final IntArrayWrapper EMPTY = new IntArrayWrapper(new int[0]);
+
+        private final int[] array;
+        private final int hash;
```

Comment:
public vs. private modifiers seem to be placed randomly.
I suggest just to use default (package-private) visibility for all, similar to other nested classes in this file.